### ISSUE TITLE
Seems like it causes some stuttering of the mouse if you run the tool multiple times.

### ISSUE THREAD

**Issue by LiamKarlMitchell** [LiamKarlMitchell/InhibitWindowsEvents#1]
Maybe Sleep(0) is better to use?

Could prevent it running multiple times too.

**Comment by LiamKarlMitchell** [comment]
Fixed by using Sleep(0) for now.

**Comment by tykavanaugh** [comment]
Thank you! I'm surprised more people aren't using this, it's incredibly useful to be able to use buttons 4 and 5 for other things.

**Comment by LiamKarlMitchell** [comment]
Yes so much better :D I really should fix this up to run as a system tray app.
